app/dashboard/src/components/AriaComponents/Inputs/ResizableInput/ResizableContentEditableInput.tsx

```diff
@@ -100,20 +110,26 @@ export const ResizableContentEditableInput = forwardRef(function ResizableConten
     defaultValue,
   })
 
-  const {
-    base,
-    description: descriptionClass,
-    inputContainer,
-    textArea,
-    placeholder: placeholderClass,
-  } = variants({
+  const styles = variants({
     invalid: fieldState.invalid,
     disabled: isDisabled || formInstance.formState.isSubmitting,
     variant,
     rounded,
     size,
   })
 
+  useEffect(() => {
```

Comment:
let's replace it with `useAutoFocus`: https://github.com/enso-org/enso/blob/17d75ffdfa764b72b715908bed69ba4c06bd66f5/app/dashboard/src/hooks/autoFocusHooks.ts#L27

Comparing to your snippet, this hook will properly move focus even if we set `autfocus` on dialog close (it'll fight the default focus restore behavior)